rosco-manifests/src/main/java/com/netflix/spinnaker/rosco/manifests/TemplateUtils.java

```diff
@@ -47,4 +52,39 @@ public Path downloadArtifactToTmpFile(Artifact artifact) throws IOException {
 
     return path;
   }
+
+  public static class BakeManifestEnvironment {
+    @Getter
+    final private Path stagingPath = Paths.get(System.getProperty("java.io.tmpdir"), UUID.randomUUID().toString());
+
+    public BakeManifestEnvironment() {
+      stagingPath.toFile().mkdirs();
```

Comment:
Might be worth a `log.debug` here.

Reply:
:+1:

Reply:
Upgraded to `warn` on failure

Reply:
My thinking was if it doesn't fail, but isn't obvious where it was created (or isn't where one expected), they could use debug logging to troubleshoot.